### PR TITLE
Optimize area form prototype

### PR DESCRIPTION
**Pull request by itkg-nanne**
[OO-FEATURE] enhance area form in template, node and area

### CODE REVIEW

File: Backoffice/Form/Type/NodeType.php [hunk 1]
```diff
@@ -50,6 +49,7 @@ public function __construct(
         $this->nodeManager = $nodeManager;
         $this->templateRepository = $templateRepository;
         $this->siteRepository = $siteRepository;
+        $this->areaContainerRepository =
```
Comment (alavieille) [inline Backoffice/Form/Type/NodeType.php]: you can removed this line

File: BackofficeBundle/Resources/config/form.yml [hunk 2]
```diff
@@ -72,6 +72,7 @@ services:
             - %open_orchestra_model.document.template.class%
             - %open_orchestra_model.document.area.class%
             - @translator
+            - @object_manager
```
Comment (alavieille) [inline BackofficeBundle/Resources/config/form.yml]: where object manager is used ?

File: Backoffice/Tests/Form/Type/NodeTypeTest.php [hunk 3]
```diff
@@ -18,12 +18,14 @@ class NodeTypeTest extends AbstractBaseTestCase
     protected $nodeClass = 'nodeClass';
     protected $areaClass = 'areaClass';
     protected $translator;
+    protected $objectManager;
```
Comment (alavieille) [inline Backoffice/Tests/Form/Type/NodeTypeTest.php]: you can remove ojbectManager

**Comment by itkg-nanne** [comment]
update done, @alavieille

**Comment by alavieille** [comment]
:+1:

**Pull request merged**